src/styles/styles.js

```diff
@@ -2297,29 +2297,45 @@ function getZoomCursorStyle(isZoomed, isDragging) {
  * @param {Number} imgHeight
  * @param {Number} zoomScale
  * @param {Number} containerHeight
+ * @param {Number} containerWidth
  * @return {Object}
  */
-function getZoomSizingStyle(isZoomed, imgWidth, imgHeight, zoomScale, containerHeight) {
+function getZoomSizingStyle(isZoomed, imgWidth, imgHeight, zoomScale, containerHeight, containerWidth) {
     if (imgWidth === 0 || imgHeight === 0) {
         return {
             height: isZoomed ? '250%' : '100%',
             width: isZoomed ? '250%' : '100%',
         };
     }
+    const top = `${Math.max((containerHeight - imgHeight) / 2, 0)}px`;
+    const left = `${Math.max((containerWidth - imgWidth) / 2, 0)}px`;
+
+    // Return different size and offset style based on zoomScale and isZoom
     if (isZoomed) {
+        if (zoomScale > 1) {
```

Comment:
Would you please note what's going on in the UI in each of these cases?

Reply:
I added the comments for each styling logic.
And noticed that my styling was not correct in the previous commit.
So, fixed it too. (when `isZoom === false && zoomScale < 1`)
Please review and let me know if something is still not clear.
Thank you

Reply:
That's perfect. Thanks!